src/content/community/releases/_index.en.md

```diff
@@ -7,6 +7,8 @@ weight = 40
 
 ## v0.17.0
 
+* The Globalnet controller now employs Kubernetes leader election to ensure proper continuity during fail-over.
```

Comment:
```suggestion
* The Globalnet controller now employs Kubernetes leader election to ensure proper continuity during fail-over to avoid potential race conditions.
```